src/client/src/utils/constants.ts

```diff
@@ -80,7 +85,8 @@ const EXTENSION_MODULES = {
   TELEMETRY: "Telemetry",
   VALIDATOR: "Validator",
   VSCODEUI: "VSCodeUI",
-  DEPENDENCYCHECKER: "DependencyChecker"
+  DEPENDENCYCHECKER: "DependencyChecker",
+  CORETS: "CoreTSModule"
 };
 
 // Define extension commands here that should be received from the extension
```

Comment:
Should remove `EXTENSION_COMMANDS.GET_PORT`

Reply:
PR updated :)